### PR TITLE
[fix](security): Use SecureRandom instead of Random, because it provides better security

### PR DESCRIPTION
**Pull request by xutaoustc**
Use SecureRandom instead of Random, because it provides better security

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/cooldown/CooldownConfHandler.java [hunk 1]
```diff
@@ -78,7 +79,7 @@ private void updateCooldownConf(List<CooldownConf> confToUpdate) {
             if (replicas.isEmpty()) {
                 continue;
             }
-            Random rand = new Random(System.currentTimeMillis());
+            Random rand = new SecureRandom();
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/cooldown/CooldownConfHandler.java:82]: Not need a seed any more???

Comment (xutaoustc) [reply fe/fe-core/src/main/java/org/apache/doris/cooldown/CooldownConfHandler.java:82]: yes, securerandom can generate a more safe seed

**Comment by bigben0204** [comment]
LGTM

**Review by bigben0204 — APPROVED** [review]

**Comment by yiguolei** [comment]
run buildall

**Review by bigben0204 — APPROVED** [review]

**Review by yiguolei — APPROVED** [review]
LGTM

**Comment by yiguolei** [comment]
run feut

**Review by yiguolei — APPROVED** [review]
LGTM

**Comment by liaoxin01** [comment]
@xutaoustc , SecureRandom seems to cause performance issues. What's the main purpose of switching to using SecureRandom?

**Pull request merged**